packages/yuudachi/src/commands/moderation/reference.ts

```diff
@@ -16,6 +15,9 @@ export default class extends Command<typeof ReferenceCommand> {
 	): Promise<void> {
 		await interaction.deferReply({ ephemeral: true });
 
+		// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+		const caseId = args.case?.case ?? args.report.case;
```

Comment:
Bc of the way args are made, `args.case` "can't" be null, so eslint doesn't like this